eq-author-api/src/validation/customKeywords/routingAnswer.js

```diff
@@ -0,0 +1,35 @@
+const {
+  ERR_ANSWER_NOT_SELECTED,
+} = require("../../../constants/validationErrorCodes");
+
+// not sure if this gonna cause problems
+const errorTypes = {
+  isNull: "Null",
+};
```

Comment:
I don't think this will cause problems because it's a null value represented as a string, so the interpreter wont treat it as a null value. If that makes sense?